legend-engine-test-server-shared/pom.xml

```diff
@@ -15,51 +15,72 @@
  limitations under the License.
 -->
 
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
```

Comment:
I created a new pom.xml for this module, but since I was also deleting a pom.xml for other module (conn-test-api), git treats it as renaming + change :) . I based this pom.xml on server module's pom.xml. If this is incorrect, then we need to change all pom.xml like server's which have this issue.